recipe/meta.yaml

```diff
@@ -1,16 +1,12 @@
-{% set version = "101" %}
+{% set version = "105" %}
 
 package:
   name: binaryen
   version: {{ version }}
 
 source:
   url: https://github.com/WebAssembly/binaryen/archive/version_{{ version }}.tar.gz
-  sha256: 5d7cdec89957549f01b7c93f080d08827c87bbd4789a34694c740d15d077c041
-  patches:
-    # this fixes an issue where "some toolchains it is necessary to #define __STDC_FORMAT_MACROS before using macros like PRIxPTR"
-    # cf: https://github.com/tensorflow/tensorflow/issues/12998#issuecomment-329179887
-    - gcc_stdmacro.patch  # [linux]
```

Comment:
@wolfv this patch no longer applies. Had asked about it here ( https://github.com/conda-forge/binaryen-feedstock/pull/46#discussion_r760696060 ), but got no response. Going to try building with it dropped. If you think it is still needed, could you please supply an updated fix?

Reply:
Alternatively just adding this to [the compiler flags]( https://github.com/conda-forge/compiler-rt-feedstock/blob/738b77edcc15e02a04200f60d579dab4a07048bf/recipe/build.sh#L24-L26 ) may be simpler/more maintainable than patching